cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/application/logging/LoggingAspect.java

```diff
@@ -60,13 +60,13 @@ private Object proceedAndLog(final ProceedingJoinPoint point) throws Throwable {
         final String securityContextDetail = securityContextDetail();
         final long startTime = System.currentTimeMillis();
         try {
-            logger.debug("Entering method call; call:[{}] {}", methodCallDetail, securityContextDetail);
+            logger.debug("Enter;[{}] {}", methodCallDetail, securityContextDetail);
             final Object result = point.proceed();
-            logger.debug("Method call returned; call:[{}] return:[{}] timeMillis:[{}] {}", methodCallDetail, result,
-                    System.currentTimeMillis() - startTime, securityContextDetail);
+            logger.debug("Return;[{}] ms:[{}] {}", result, System.currentTimeMillis() - startTime,
```

Comment:
Seeing as this change removes the method call detail, we can also remove the security context detail (this part describes the principal and team ids)

Reply:
Ive removed that extra logging as suggested, thanks fo heads up